docs/api-reference/webgl/texture-3d.md

```diff
@@ -21,15 +19,15 @@ if (Texture3D.isSupported()) {
 
 * `handle` - The underlying `WebGLTexture`
 * `target` - Always `GL.TEXTURE_3D`
-* `depth` - the number of texture layers
-* `width` - width of the layer textures
-* `height` - height of the layer textures
-* `format` - format of the layer textures
+* `width` - width of texture
+* `height` - height of textire
```

Comment:
typo: `textire`.

Again why cut: You don't feel it is valuable to emphasize that this applies to the all the texture layers to help the user understand how this object works?

Reply:
Same as above. The old descriptions are for texture arrays and don't make sense for 3D texture.